tests/ssl-opt.sh

```diff
@@ -638,6 +639,7 @@ requires_hash_alg() {
     HASH_DEFINE="Invalid"
     HAS_HASH_ALG="NO"
     case $1 in
+        MD_5):;;
```

Comment:
MD5 is always spelled MD5 without a hyphen or underscore, unlike SHA-1/SHA-2/SHA-3 which are formally spelled with a hyphen but a hyphenless variant is common.